L1Trigger/DemonstratorTools/src/codecs_vertices.cc

```diff
@@ -0,0 +1,21 @@
+
+#include "L1Trigger/DemonstratorTools/interface/codecs/vertices.h"
+
+namespace l1t::demo::codecs {
+
+  std::vector<l1t::Vertex> decodeVertices(const l1t::demo::BoardData::Channel& frames) {
+    std::vector<l1t::Vertex> vertices;
+
+    for (const auto& x : frames) {
+      if (not(x.data & 1))
+        break;
+
+      // TODO: Replace with complete implementation
+      Vertex vertex(0.0, ap_fixed<15, 6>(ap_uint<64>(x.data)(15, 1)), {});
```

Comment:
The position of the part of the word you're trying to extract, `(15,1)`, should be taken from the VertexWord data format as well.

Reply:
Done - am now using the `enum` valuess from `VertexWord`